packages/expo-crypto/src/Crypto.types.ts

```diff
@@ -0,0 +1,16 @@
+export enum Algorithm {
+  sha1 = 'SHA-1' /* (but don't use this in cryptographic applications) */,
+  sha256 = 'SHA-256',
+  sha384 = 'SHA-384',
+  sha512 = 'SHA-512',
+  md2 = 'MD2',
```

Comment:
You'll need to document which platforms support which algorithms (web does not support MD*)